main.go

```diff
@@ -41,4 +42,12 @@ func main() {
 	if err != nil {
 		log.Fatal("Could not start GlusterD Rest Server. Aborting.")
 	}
+
+	// Starting etcd daemon upon starting of GlusterD
+	etcd_start := exec.Command("/bin/etcd")
```

Comment:
I have tested this change only in sample programme. this is not my final change. will add some parameter in new PR.